<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>808Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>660</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>660</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>660</y>
      <width>148</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Display</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Config Box</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_P">
    <property name="title">
     <string>参数设置(&amp;P)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="title">
     <string>多媒体(&amp;M)</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>行车记录仪</string>
    </property>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>下发指令</string>
    </property>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
    <addaction name="action_20"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>模拟终端</string>
    </property>
    <addaction name="action14544916843"/>
   </widget>
   <addaction name="menu_P"/>
   <addaction name="menu_M"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>设置配置参数</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>查询配置参数</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>终端控制</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>设置电话本</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>车辆控制</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>拍照</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>录音</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>电话回拨</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>多媒体检索</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>单条多媒体上传</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>行车记录仪数据采集</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>行车记录仪数据上传</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>临时位置跟踪</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>位置信息查询</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>文本信息下发</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>信息点播设置</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>事件设置</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>提问下发</string>
   </property>
  </action>
  <action name="action14544916843">
   <property name="text">
    <string>14544916843</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
